<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_adiciona_atividade</class>
 <widget class="QMainWindow" name="janela_adiciona_atividade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background: #4A90E2;
}

QLabel{
	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;
	font-size: 18px;
    color: #fff;
}

#_txt_titulo{
	font-size: 20px;
}

QPushButton{
    background: #4A90E2;
    border: 2px solid #fff;
    border-radius: 14px;
    padding: 10px 20px;

	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;
	font-size: 18px;
    color: #fff;
}

QLineEdit{
    background: #ffffff;
    border: 2px solid #4A90E2;
    border-radius: 8px;
    padding: 5px 10px;

    font-family: &quot;Segoe UI&quot;;
    font-weight: bold;
    font-size: 16px;
    color: #333333;
}

QLineEdit:focus{
    border-color: #53A3FF;
    background: #E6F7FF; 
}

QLineEdit:disabled {
    background: #f0f0f0;
    border: 2px solid #cccccc;
    color: #999999;
}

QComboBox {
    background: #ffffff;
    border: 2px solid #4A90E2;
    border-radius: 8px;
    padding: 5px 10px;

    font-family: &quot;Segoe UI&quot;;
    font-weight: bold;
    font-size: 16px;
    color: #333333;
}

QComboBox:focus {
    border-color: #53A3FF;
    background: #E6F7FF;
}

QComboBox:disabled {
    background: #f0f0f0;
    border: 2px solid #cccccc;
    color: #999999;
}

/* Estilizando a seta do QComboBox */
QComboBox::drop-down {
    border: none;
    width: 25px;
}

QComboBox::down-arrow {
    image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAICAYAAADJ+H49AAAAG0lEQVR42mJk+M8ABkmEYGBg+P///zCABQBVcgQCPh8pRgAAAABJRU5ErkJggg==); /* Uma seta simples, você pode ajustar */
    width: 10px;
    height: 10px;
}

/* Estilizando a lista suspensa */
QComboBox QAbstractItemView {
    background: #ffffff;
    border: 1px solid #4A90E2;
    selection-background-color: #E6F7FF;
    selection-color: #333333;

    font-family: &quot;Segoe UI&quot;;
    font-weight: bold;
    font-size: 16px;
    color: #333333;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="_btn_adicionar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="_cb_atividade">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Atividade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_line_repeticao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Repetição</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_titulo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nova atividade:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="_cb_tipo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Tipo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>378</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
